javaparser-core/src/main/java/com/github/javaparser/ParserConfiguration.java

```diff
@@ -35,14 +40,73 @@
  * It will pick up the changes.
  */
 public class ParserConfiguration {
+    public enum LanguageLevel {
+        ANY(null, null),
+        JAVA_1_0(new Java1_0Validator(), null),
+        JAVA_1_1(new Java1_1Validator(), null),
+        JAVA_1_2(new Java1_2Validator(), null),
+        JAVA_1_3(new Java1_3Validator(), null),
+        JAVA_1_4(new Java1_4Validator(), null),
+        JAVA_5(new Java5Validator(), null),
+        JAVA_6(new Java6Validator(), null),
+        JAVA_7(new Java7Validator(), null),
+        JAVA_8(new Java8Validator(), null),
+        JAVA_9(new Java9Validator(), null),
+        JAVA_10(null, new Java10PostProcessor());
```

Comment:
Maybe we could call it JAVA_10_preview and tell that we are going to remove it later, when Java_10 is actually released

Reply:
Hmmm, yes, that's an interesting suggestion.